airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -274,15 +301,14 @@ def _try_reattach_task(self):
         )
         running_tasks = list_tasks_resp['taskArns']
 
-        running_tasks_count = len(running_tasks)
-        if running_tasks_count > 1:
-            self.arn = running_tasks[0]
-            self.log.warning('More than 1 ECS Task found. Reattaching to %s', self.arn)
-        elif running_tasks_count == 1:
-            self.arn = running_tasks[0]
-            self.log.info('Reattaching task: %s', self.arn)
+        # Check if the ECS task previously launched is already running
+        previous_task_arn = self.xcom_pull(task_ids=f"{self.task_id}_task_arn", key="ecs_task_arn")
+        self.log.info(f"Previously launched task = {previous_task_arn}")
```

Comment:
Nit: It probably adds noise to log this if we're not going to use it. I think the log line inside the if statement below is sufficient.

Reply:
I will remove it, thanks!